hbase-client/src/main/java/org/apache/hadoop/hbase/client/AsyncNonMetaRegionLocator.java

```diff
@@ -50,6 +50,7 @@
 import java.util.concurrent.TimeUnit;
 import org.apache.commons.lang3.ObjectUtils;
 import org.apache.hadoop.hbase.CatalogFamilyFormat;
+import org.apache.hadoop.hbase.CatalogReplicaMode;
```

Comment:
We just add an import without touching the actual code?

Reply:
CatalogReplicaMode is moved from org.apache.hadoop.hbase.client to org.apache.hadoop.hbase, this change reflects the move.